Upstream-Status:Inappropriate [debian patch]

From: Niko Tyni <ntyni@debian.org>
Subject: Fix CPANPLUS test failures when HOME doesn't exist
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=52988
Bug-Debian: http://bugs.debian.org/577011
Origin: upstream

The Debian autobuilders are configured with a non-existing $ENV{HOME},
triggering a bug in CPANPLUS that causes test failures.

Fix from CPANPLUS-0.9001.

---
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
index 27d2abc..8475c36 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
@@ -5,7 +5,7 @@ use strict;
 use CPANPLUS::Error;
 use CPANPLUS::Internals::Constants;
 
-use Cwd qw[chdir];
+use Cwd                         qw[chdir cwd];
 use File::Copy;
 use Params::Check               qw[check];
 use Module::Load::Conditional   qw[can_load];
-- 
tg: (a508b62..) fixes/cpanplus-without-home (depends on: upstream)
